README.md

```diff
@@ -34,6 +34,13 @@ TCEFORM.pages.backend_layout.disabled = 1
 TCEFORM.pages.backend_layout_next_level.disabled = 1
 ```
 
+### Note
+
+This extension comes with a default empty backend_layout configuration (see `Configuration/PageTs/main.tsconfig`) for
+TYPO3's default "empty" page types (like Shortcut, External Link, etc.) to ensure valid database entries for every page
+type (even the ones without any content).
```

Comment:
i would write "to render proper Backend Layouts)"
and
"you have to import this PageTS"